src/components/EventCard.tsx

```diff
@@ -88,22 +108,23 @@ const EventCard = ({
             {description}
           </p>
         </div>
-
-        <div className="flex flex-row flex-wrap">
-          {status === Status.Live ? (
-            <button className="button-base-red px-4 my-2">Vote</button>
-          ) : (
-            <>
-              <button className="button-base-white px-2 my-2 mr-5">
-                Unregister
-              </button>
-              <button className="button-base-red px-4 my-2">See More</button>
-            </>
-          )}
+       
+          <div className="flex flex-row flex-wrap">
+            {status === Status.Live ? (
+              <button className="button-base-red px-4 my-2">Vote</button>
+            ) : (
+              <>
+                <button onClick={toggleReg} className={regButtonStyle}>
+                  {isRegistered ? "Unregister" : "Register"}
+                </button>
+                <button className="button-base-red px-4 my-2">See More</button>
```

Comment:
Can we also make the `See more` button the same size as the `Unregister` button for consistency? (Can also ask the designer aka jawsh)